pep-9999.rst

```diff
@@ -717,6 +728,30 @@ Maybe::
 Avoid ``Union`` return types, since they require ``isinstance()`` checks.
 Use ``Any`` if necessary.
 
+NamedTuple and TypedDict
+------------------------
+
+Use the class-based syntax for ``typing.NamedTuple`` and
+``typing.TypedDict``, following the Classes section of this style guide.
+
+Yes::
+
+    from typing import NamedTuple, TypedDict
+    class Point(NamedTuple):
+        x: float
+        y: float
+        def distance(self, other: Point) -> float: ...
```

Comment:
nit: I'd take out this method, to make the definitions in the "Yes" and "No" sections exactly match, to drive home the point that the class-based syntax should be used even when the same thing can be expressed using the function-based syntax.